src/Policies.cpp

```diff
@@ -146,6 +143,14 @@ std::vector<Policy *> PolicyManager::load(const YAML::Node &policy_yaml)
                 throw PolicyException(fmt::format("unable to instantiate tap '{}': {}", tap_name, e.what()));
             }
         }
+
+        InputEventProxy *input_event_proxy = nullptr;
+        try {
+            input_event_proxy = input_ptr->add_event_proxy(tap_filter);
+        } catch (ConfigException &e) {
+            throw PolicyException(fmt::format("invalid input filter config: {}", e.what()));
```

Comment:
should this error message be broader? "unable to create event proxy"? not sure.

Reply:
done, please check